pkg/apis/app/v1alpha1/application_types.go

```diff
@@ -38,6 +38,7 @@ type ApplicationSpec struct {
 	Version string `json:"version,omitempty"`
 
 	// Description is a brief string description of the Application.
+	// Should contain plain text or plain-text-friendly markup (eg. markdown).
```

Comment:
The idea of this is a simple string description rather than something long form. Here are examples from other platforms:

node.js (forever app):
> A simple CLI tool for ensuring that a given node script runs continuously (i.e. forever)

composer/packagist (guzzle library):
> Guzzle is a PHP HTTP client library

helm (artifactory chart):
> Universal Repository Manager supporting all major packaging formats, build tools and CI servers.

If we look at something like Ubuntu snaps there's a one like summary and a multi-line text description. One is meant for a sentence and the other a paragraph.

This is the idea for description. Why would it being from being in Markdown?

Reply:
Agreed, as suggested above, I think this should be plain-text only

Reply:
SG. I'm reverting this change.